source/Files.cpp

```diff
@@ -545,10 +546,9 @@ void Files::Write(FILE *file, const string &data)
 
 
 
-void Files::LogError(const string &message)
+void Files::LogErrorToFile(const string &message)
 {
-	lock_guard<mutex> lock(errorMutex);
-	cerr << message << endl;
+	lock_guard<mutex> lock(errorFileMutex);
```

Comment:
Since this function is already guarded by the mutex in LogError, it should be safe to remove it from here.

Reply:
Agreed